src/shared/condition.c

```diff
@@ -157,6 +158,7 @@ static OrderOperator parse_order(const char **s) {
                 [ORDER_GREATER_OR_EQUAL] = ">=",
                 [ORDER_LOWER] = "<",
                 [ORDER_GREATER] = ">",
+                [ORDER_EQUAL_DOUBLE] = "==",
```

Comment:
i am not sure i like this redundancy that much. The `==` thing is a C thing, and got inherited into other languages I guess. But shell for example (i.e. `test`) doesn't do that, and I am not sure we should. Can you make a strong point why it is worth the redundancy?

I think I don't like this. At the absolute minimum we should not show the redundancy in the man page, since it' just confusing, people might assume "=" and "==" are different things...

But I actually think we should simply not do this at all, and just drop the == thing.

Reply:
I don't mind either way - I added it as it was requested here: https://github.com/systemd/systemd/pull/19941#discussion_r657013360